examples/python/controlnet/main.py

```diff
@@ -0,0 +1,141 @@
+#!/usr/bin/env python3
+"""
+Example running ControlNet conditioned on Canny edges.
+
+Based on https://huggingface.co/docs/diffusers/using-diffusers/controlnet.
+
+Run
+```sh
+python 3 examples/python/controlnet/main.py
+```
+"""
+from __future__ import annotations
+
+import argparse
+import os
+
+import cv2
+import numpy as np
+import PIL.Image
+import requests
+import rerun as rr
+import torch
+from diffusers import (
+    AutoencoderKL,
+    ControlNetModel,
+    StableDiffusionXLControlNetPipeline,
+)
+
+
+def controlnet_callback(
+    iteration: int, timestep: float, latents: torch.Tensor, pipeline
+) -> None:
+    rr.set_time_sequence("iteration", iteration)
+    image = pipeline.vae.decode(
+        latents / pipeline.vae.config.scaling_factor, return_dict=False
+    )[0]
+    image = pipeline.image_processor.postprocess(image, output_type="np")[0]
+    rr.log("output", rr.Image(image))
+    rr.log("latent", rr.Tensor(latents[0], dim_names=["channel", "height", "width"]))
```

Comment:
Would this work instead?
```suggestion
    rr.log("latent", rr.Tensor(latents.squeeze(), dim_names=["channel", "height", "width"]))
```

Reply:
Replaced with `[0]` with `.squeeze()` whenever possible.